static/src/javascripts/projects/commercial/modules/brazeBanner.js

```diff
@@ -162,13 +162,25 @@ const getMessageFromBraze = async (apiKey, brazeUuid) => {
 };
 
 const maybeWipeUserData = async (apiKey, brazeUuid) => {
-    if (!brazeUuid && hasCurrentBrazeUser()) {
-        appboy = await import(/* webpackChunkName: "braze-web-sdk-core" */ '@braze/web-sdk-core');
-
-        appboy.initialize(apiKey, SDK_OPTIONS);
+    const userHasLoggedOut = !brazeUuid && hasCurrentBrazeUser();
+    const slotNames = ['Banner','EndOfArticle']
 
+    if (userHasLoggedOut) {
         try {
-            appboy.wipeData();
+            if (apiKey){
+                appboy = await import(/* webpackChunkName: "braze-web-sdk-core" */ '@braze/web-sdk-core');
+                appboy.initialize(apiKey, SDK_OPTIONS);
+                appboy.wipeData();
+            }
+
+            // DCR has an implementation of LocalMessageCache but Frontend does not
+            // We should still wipe the cache from Frontend if the user logs out
+            const localStorageKeyBase = 'gu.brazeMessageCache'
+            slotNames.forEach(slotName => {
+                const key = `${localStorageKeyBase}.${slotName}`
+                storage.local.remove(key);
```

Comment:
It doesn't look like we're importing `storage` here? You could either import it from `@guardian/libs` or use `localStorage.removeItem(key)` directly.

Reply:
Ah yes, good spot. I'll sort it out.